Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "prometheus"
+version = "0.1.0"
+keywords = ["prometheus", "metrics"]
+
+[features]
+default = []
+dev = ["clippy"]
+
+[dependencies]
+protobuf = "1.0"
+quick-error = "0.2"
+clippy = {version = "*", optional = true}
```

Comment:
I have no idea if crates.io also filters optional dependencies the same way (I guess so), but in that case you can turn it into a `>= 0.0.0` semver requirement.